browsers/safari.json

```diff
@@ -173,6 +173,11 @@
           "status": "current",
           "engine": "WebKit",
           "engine_version": "610.1.28"
+        },
+        "14.1": {
+          "release_notes": "https://developer.apple.com/safari/technology-preview/release-notes/",
```

Comment:
We can’t use the TP release notes URL here. Instead we need to wait until the 14.1 beta is released. And I think that’s not going to happen until several weeks from now  Or months, really.

Reply:
> @sideshowbarker Does MDN not consider STP to be a tracked version?

For Safari, unfortunately our current policy is to not track STP versions, but instead only major releases. Since for Safari, major releases only happen twice a year, that’s not super optimal. But anyway, that’s the state of things.

> If I need to wait until there's a version shipping as beta I'll wait until then to revisit this PR of course.

Yeah I think that in this case we unfortunately need to wait

Reply:
Sounds good - no worries.